### PR TITLE
Feature/mc 92 data transformation

### PR DESCRIPTION
**Pull request by kennyevil**
This is a big one unfortunately. It covers MC-92 for the data transformation and MC-116 for the admin upload tool.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -94,6 +94,16 @@
     get '/html/select-location', to: 'html#select_location'
     get '/html/supplier-detail', to: 'html#supplier_detail'
     get '/html/download-the-supplier-list', to: 'html#download_the_supplier_list'
+    unless Rails.env.production? # not be available on production environments yet
```
Comment (iceraluk) [inline config/routes.rb]: Development environment has 'production' Rails.env so you will not be able to test this on Dev

File: db/schema.rb [hunk 2]
```diff
@@ -10,22 +10,13 @@
...
 
-ActiveRecord::Schema.define(version: 2019_04_12_144942) do
+ActiveRecord::Schema.define(version: 2019_05_21_111111) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
   enable_extension "postgis"
 
-  create_table "facilities_management_buildings", id: false, force: :cascade do |t|
```
Comment (iceraluk) [inline db/schema.rb]: Should this be removed by this PR? @tariqhamid what do you think

Comment (kennyevil) [reply db/schema.rb]: Good catch. Something keeps on getting messed up with the schema between PRs with tables getting dropped like this. I'll try to fix it now.

Comment (iceraluk) [reply db/schema.rb]: I get the same. think we might be missing some migrations.

Comment (kennyevil) [reply db/schema.rb]: Talked to @tariqhamid and I think we should put in a separate PR to add the missing migrations.

**Comment by kennyevil** [comment]
@iceraluk Could you give this another look?

**Pull request merged**